base_config.yml

```diff
@@ -6,3 +6,7 @@ DOCKER_TIMEOUT: 600
 # Common build_vars
 MODD_VERSION: &MODD_VERSION 0.5
 TASKFILE_VERSION: &TASKFILE_VERSION 3.2.2
+
+base_platforms: &base_platforms
+  - linux/amd64
```

Comment:
Wouldn't it be easier to have `linux/amd64` as base config and add the arm platform in the images step by step?